src/middleware/environment.js

```diff
@@ -131,5 +133,32 @@ module.exports = function environmentMiddleware(argv) {
 
     return resp
   }
+
+  // if there is no arapp.json and the command is not init default to the "global" config
+  // designed for the dao commands including dao acl
+  if (!module && !runsInCwd) {
+    const defaultEnvironments = require('../../config/environments.default')
+    const defaultNetworks = require('../../config/truffle.default')
+
+    let { environment, apm } = argv
+    let env
+
+    if (!environment) {
+      env = defaultEnvironments['aragon:local']
+    } else {
+      env = defaultEnvironments[environment]
```

Comment:
Could condense this to `const env = defaultEnvironments[environment || 'aragon:local']`